### ISSUE TITLE
Android: Espeak language intermittently changes to US English even with system/Espeak language set to UK

### ISSUE THREAD

**Issue by codeofdusk** [espeak-ng/espeak-ng#970]
When navigating the system using Espeak with TalkBack (both Espeak and system language set to UK English), Espeak suddenly and intermittently changes to US English. Opening TTS settings without making any other changes causes Espeak to switch back to UK.

I can reproduce this both in a source build of ab11439 and in the 1.48 Play Store version.

**Comment by MH140000** [comment]
Yes. I have the same problem with Persian. espeak is switched to english US.
I have to use the auto tts utility.
